A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js

```diff
@@ -258,7 +265,7 @@ var A320_Neo_LowerECAM_APU;
                 if (n < 10) {
                     return 10;
                 } else if(n <14){
```

Comment:
That formatting should be fixed while you're at it

Reply:
done